### ISSUE TITLE
Data Hooke: Donor Entity Page

### ISSUE THREAD

**Issue by rosibaj** [icgc-argo/platform-ui#1863]

**Comment by akachru-github** [comment]
Clinical API should be updated with the logic to provide a value like "NA". Business logic should not be added to the FE to interpret a return like "NULL" for example. @joneubank , @mistryrn , @demariadaniel  to connect on this to agree how this will work.

**Comment by akachru-github** [comment]
As first step:
- Read what we get and render in the table. 
- To complete this, identify mandatory fields that will be displayed regardless of data. @edsu7 can help with identifying these.

**Comment by Buwujiu** [comment]
Question for @edsu7 @lindaxiang :
The mockup shows less Data Category than what's on argo qa right now: 
![Screen Shot 2022-11-11 at 1.06.32 PM.png](https://images.zenhubusercontent.com/5c9a3377f234b2000189633c/3d55e050-6312-42f7-a791-0578be8ba715)
Do we want to have all the data categories on the file cards?

**Comment by lindaxiang** [comment]
@Buwujiu , Data Categories in QA are a bit messy. In Prod, we should not have that many categories, maybe we can make the info on file cards to be consistent with facet in the file repository.

**Comment by Buwujiu** [comment]
> @Buwujiu , Data Categories in QA are a bit messy. In Prod, we should not have that many categories, maybe we can make the info on file cards to be consistent with facet in the file repository.

@lindaxiang do you mean we should have a fixed list of Data Category on the available file cards? would this. be easier for users?

**Comment by lindaxiang** [comment]
@Buwujiu no, I meant the dynamic list of Data Categories are preferable.

**Comment by Buwujiu** [comment]
> @Buwujiu no, I meant the dynamic list of Data Categories are preferable.

@lindaxiang is it possible that a donor might not have files in data categories? Say we have 8 categories in total, but some donors might only have files in 5 of those categories?
